fury/actor.py

```diff
@@ -2553,23 +2573,26 @@ def sdf(centers, directions=(1, 0, 0), colors=(1, 0, 0), primitives='torus',
         The orientation vector of the SDF primitive.
     primitives : str, list, tuple, np.ndarray
         The primitive of choice to be rendered.
-        Options are sphere and torus. Default is torus.
+        Options are sphere, torus and ellipsoid. Default is torus.
     scales : float
         The size of the SDF primitive
 
     Returns
     -------
     vtkActor
     """
-
+    print("the size of centres is ---", centers.shape)
     prims = {'sphere': 1, 'torus': 2, 'ellipsoid': 3}
 
     verts, faces = fp.prim_box()
+    print("The size of verts and faces is ---", verts.shape, faces.shape)
     repeated = fp.repeat_primitive(verts, faces, centers=centers,
                                    colors=colors, directions=directions,
                                    scales=scales)
+   # print("Repeated is------", repeated)
 
     rep_verts, rep_faces, rep_colors, rep_centers = repeated
+    print("The size of following is : rep_verts", rep_verts.shape, "rep_faces", rep_faces.shape, "rep_colors", rep_colors.shape, "rep_centers", rep_centers.shape)
```

Comment:
many `print` to remove

Reply:
Hello @skoudoro 
I apologize for the mistakes and the inconvenience caused. I have created a fresh pull request with cleaned code and included unit tests. 
Thank You